txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+      as it evaporates.
+
+4. Move vials to vacuum chamber and vacuum for an additional 2 hours..
+
+5. Store in -20 degC.
+
+**Note:** Remaining POPC in chloroform, cholesterol, and  Lyss-Rho-PE in chloroform can be stored at -20 degC in the glass vial with PTFE caps and sealed parafilm.
+
+## Step 1: Preparation of the oil containing lipids
+
+1. Mix mineral oil by gentle inversion before use 
+
+2. Place 0.5 mL of mineral oil into each of the vials.
+
+3. Incubate at 60 degC  for 10 min
+
+4. Vortex for 10 mins
+
+5. Incubate vials for 3 hrs at 60 degC.
+
+6. Wrap up the top of the vials with aluminum foil and seal with parafilm.
+
+7. Sonicate in an heated water bath for 30 minutes,at 60 degC .
+
+    a. **Note:** If a sonicator is unavailable then repeat 60 degC
+    incubation and vortexing until the lipid is completely dissolved.
+
+## Step 2: Self-assembly of the Liposomes by centrifugation
+
+1. Place 225 μL of centrifuge buffer (100 mM HEPES + 200 mM glucose,
+   pH 8) into a labeled eppendorf tube.
+
+    a. **Note**: When IVTL systems are used, the outer solution should
+    also contain the small molecular weight components of the IVTL
+    system.
+
+2. Add in 30 uL of inner solution (100 mM HEPES, 200 mM sucrose pH 8,
+   and, if needed, 2 uM of a water-soluble fluorescent dye like HPTS
+   or calcein.) to 500 uL of suspended lipid in oil from Step
+   1. Vortex for 30 s. Equilibrate for 10 min at 4 degC.
+
+    * **Note**: When IVTL systems are used, buffer is omitted. IVTLs
+    come with their own buffers.
+
+3. Add the emulsion on top of the 225 μL centrifuge buffer by
+pipetting against the wall of the tube, wait at least 1 minute for the
+interface to stabilize and flatten between the emulsion and buffer
+
+4. Centrifuge at 18000 rcf at 4 degC  for 15 min  
+
+    * **Note:** If the top phase (oil) is clear, that suggests that
+      droplets have passed into the bottom buffer, becoming vesicles.
+
+5. Carefully remove as much mineral oil as possible with a
+   gel-loading-pipette-tip from the top.
+
+    * **Note: **The goal is to remove as much oil up to the interface
+    without disturbing the buffer below.
+
+6. Place 225 μL of wash buffer (100 mM HEPES + 250 mM glucose, pH 8)
+   into a labeled eppendorf tube.
+
+7. Add the centrifuge buffer (where the vesicles should have formed)
+   to eppendorf tubes with wash buffer. Make sure to use a new tip to
+   avoid contamination.
+
+    * **Note:** An alternative method is to open the eppendorf tube
+    and use a 21-gauge needle to punch a hole at the bottom of the
+    eppendorf tube. Remove the needle and the close the lid to allow
+    the buffer solution to drip out.
+
+8. Centrifuge at 12000 rcf at 4 degC  for 5 min
+
+9. Transfer 225uL from the bottom of the eppendorf tubes with wash
+   buffer into a new labeled eppendorf tube.
+
+10. Add 10 μL of solution to 10 μL of final buffer (50 mM HEPES + 300
```

Comment:
I think this should be removed (it was struck out in the Google Doc, but was transferred as normal test).

Reply:
Step 10 can be removed